Fluent/Post_Process.ipynb

```diff
@@ -6,127 +6,63 @@
    "source": [
     "# CFD Results Post-Process & Validation\n",
     "\n",
-    "< copy and paste the two below paragraphs on top of your Post_Process file. Remove this guideline after that. >\n",
-    "\n",
     "**The final step for the development of a CFD simulation for a problem of interest is to process and examine the validity of the obtained numerical results (a.k.a post-processing and validation step). These steps requires an in-depth engineering knowledge about the fundamentals of the physical problem of interest. Thus, the value of investing time in understanding the fundamental physics of the flow, once again is highlighted here. This process is done starting from general inspection on CFD simulation results and ends with more detail validation process.**\n",
```

Comment:
Before start editing this section, I highly recommend you to take a close look at the content of the Post Processing file that we finalized for the Open Foam part. Lots of the analysis and developed sentences can be used here. However, first you need to read it and understand the structure of it.